### PR TITLE
Retry L1 transaction broadcast

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

Currently, we do not retry broadcasting transactions to the L1 from the host. We error out after the first failure. This is not resilient.

### What changes were made as part of this PR:

Functional.

- Sets timeout for all L1 client requests
- Adds retries for L1 transaction broadcast

### What are the key areas to look at

- ...

### :rotating_light: Definition of Done :rotating_light:
- [ ] Deployed into dev-testnet 
- [ ] Passes tests on dev-testnet
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: go/host/node/node.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ const (
 	apiNamespaceObscuroScan = "obscuroscan"
 	apiNamespaceNetwork     = "net"
 	apiNamespaceTest        = "test"
+
+	l1TxRetriesRollup = 3
```
Comment (tudor-malene) [inline go/host/node/node.go:43]: some comments would be helpful here describing what these constants are and how much time that means

**Review by tudor-malene — APPROVED** [review]
lgtm. 
a few comments would be helpful

File: go/host/node/node.go (hunk 1 above)
Comment (joeldudley) [reply go/host/node/node.go:43]: I'll add a comment. It's hard to say accurately with timing, as on top of the pauses, there's the duration of the request itself.

Comment (tudor-malene) [reply go/host/node/node.go]: of course. just ballpark.

Comment (joeldudley) [reply go/host/node/node.go]: 👍

**Pull request merged**